sidebars.js

```diff
@@ -14,22 +14,8 @@ module.exports = {
     label: 'Installation',
     link: {type: 'doc', id: 'installation/installation',},
     items:[
-      {
-        type: 'category',
-        label: 'Installation Requirements',
-        link: {type: 'doc', id: 'installation/installation-requirements/installation-requirements',},
-        items: ['installation/installation-requirements/resource-profiling'],
-      },
-      {
-        type: 'category',
-        label: 'Installation Options',
-        link: {type: 'doc', id: 'installation/install-options/install-options',},
-        items: [
-          'installation/install-options/server-config',
-          'installation/install-options/agent-config',
-          'installation/install-options/how-to-flags',
-        ],
-      },
+      'installation/installation-requirements',
+      'installation/install-options',
```

Comment:
I feel like we should take the "installation" prefix off these, since they're already in the "installation" section. 
```suggestion
      'installation/requirements',
      'installation/script-options',
```

Additionally, the current "install options" page contains a mix of install script options, basic `k3s` command use, and basic info on config file support. I feel like that page should have just info on the install script, and the other bits should go on their own pages.

Reply:
First) I removed the prefixes on installation requirements

Second) I have reworked both the reference section and the install-options page. It is now called `Configuration` and is basically a swap with the old `how to flags` section. Now the Reference section hold the long table of information, including a broken out `K3s Binary Tools` page and `Environmental Variables` page.

I actually like how all 3 configuration options were under one page, so I kept that.